spec/bourbon/settings/bourbon_get_setting_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "spec_helper"
+
+describe "_bourbon-get-setting" do
+  before(:all) do
+    ParserSupport.parse_file("settings/bourbon-get-setting")
+  end
+
+  context "gets the modular-scale-base setting" do
+    it "and returns the default value" do
+      expect(".test-1").to have_rule("content: 1em")
+    end
+  end
+
+  context "gets the rails-asset-pipeline setting" do
+    it "and returns the user-overridden value" do
+      expect(".test-2").to have_rule("content: true")
+    end
+  end
+
+  context "called from the font-face mixin" do
+    it "outputs user-overridden font file formats" do
+      ruleset = 'font-family: "source-sans-pro"; ' +
+                'font-style: normal; ' +
+                'font-weight: normal; ' +
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.